test/activation.jl

```diff
@@ -127,6 +130,12 @@ end
         @test elu(-4) ≈ (exp(-4) - 1)
     end
 
+    @testset "mish" begin
+        @test mish(-5) ≈ -0.033576237730161704
+        @test mish(9) == 9*tanh(log(1 + exp(9)))
+        xs = Float32[1 2 3; 1000 2000 3000]
+        @test typeof(mish.(xs)) == typeof(xs)
```

Comment:
I think you're missing an `end` here. :)

Reply:
Fixed. Thanks!